client/views/edit/edit.html

```diff
@@ -10,7 +10,7 @@ <h1 class="formtitle">Edit</h1>
 		<div class="inputcontainer" style="text-align: center;">
```

Comment:
Let's fix the formatting here.

P.S. You can do it in every html template, just remember to do it in a separate commit (and call it something like `Fixed formatting`).

Reply:
Updated the formatting

Reply:
Updated the formatting